tmt/steps/prepare/install.py

```diff
@@ -234,6 +234,21 @@ class InstallRpmOstree(InstallBase):
     package_manager = "rpm-ostree"
     copr_plugin = "dnf-plugins-core"
 
+    def sort_packages(self):
+        self.recommend_packages = []
+        self.required_packages = []
+        for package in self.repository_packages:
+            try:
+                msg = self.guest.execute(f"rpm -q --whatprovides {package}")
+            except tmt.utils.RunError:
+                if self.skip:
+                    self.recommend_packages.append(package)
+                else:
+                    self.required_packages.append(package)
+                continue
+            # Do nothing
+            self.warn(f"Package: '{package}' already provided '{msg.stdout}'")
```

Comment:
Would a `self.info()` or `self.debug()` more appropriate when a required/recommended package is already installed? You can also move this line right after `self.guest.execute` to make the code a bit more straightforward.